src/main/kotlin/rys/ajaxpetproject/controller/ChatController.kt

```diff
@@ -0,0 +1,39 @@
+package rys.ajaxpetproject.controller
+
+import jakarta.validation.Valid
+import org.bson.types.ObjectId
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import rys.ajaxpetproject.model.MongoChat
+import rys.ajaxpetproject.service.ChatService
+
+@RestController
+@RequestMapping("/chats")
+class ChatController(val chatService: ChatService) {
+
+    // Create a new Chat
+    @PostMapping("/")
+    fun createChat(@RequestBody mongoChat: MongoChat): ResponseEntity<MongoChat> =
+        ResponseEntity(chatService.createChat(mongoChat), HttpStatus.CREATED)
+
+
```

Comment:
remove please the extra line. Try to keep all your code in the same style

Reply:
Please, check another places as well

Reply:
done